src/SourceCompile/Compiler.cpp

```diff
@@ -303,7 +303,7 @@ bool Compiler::createFileList_() {
         }
         ofs.close();
       } else {
-        std::cout << "Could not create filelist: " << fileList << std::endl;
+        std::cerr << "Could not create filelist: " << fileList << std::endl;
```

Comment:
There are many non-related changes in this file. Try comparing while discarding spaces and the changes might be more obvious. The function body moved left because of the early out which shows up as lot more changes than their really are.